script/Ciel/Day 9/09_01_CIEL09_8.txt

```diff
@@ -537,7 +537,7 @@ I can't bring myself to admit it.
 -- Page 730, Offset 24033.
 -- シエル先輩を気遣っての事じゃない。俺は俺の為に、その事実から目を背けたかっただけ。
 Not because I'm trying to avoid hurting Ciel-senpai,
- but for purely selfish reasons――I just don't want to have to face that reality.
+ but for purely selfish reasons――I just don't want to face that reality.
```

Comment:
I think I might prefer this one as-is, just because "I don't want to _be forced to_ face that reality" makes it sound a bit more unpleasant to him

Reply:
Prefer the change, I feel it more strongly conveys the selfishness. 
'I don't want to face that reality' - He has a choice and he chooses no.
'I don't want to have to face it'/'I don't want to be forced to' - He doesn't want to not have a choice.